docs/lifecycle/lifecycle.rst

```diff
@@ -24,12 +24,18 @@ Project States
 +---------------+-------------------------------------------------------------+
 | Project State | State Summary                                               |
 +===============+=============================================================+
-| Non-LFN       | Project does not exist or exists outside of LFN.            |
+| none          | Project does not exist or exists outside of LFN.            |
 +---------------+-------------------------------------------------------------+
-| Non-TAC       | Project is admitted to the LFN but does not have a          |
-|               | representative on the TAC.                                  |
+| Sandbox       | Project is admitted to LFN but does not have direct funding |
+|               | from LFN.  The intent is to enable new projects to gain     |
+|               | visibility and participate in the LFN with minimal impact   |
+|               | on existing projects until they are ready for a subsequent  |
+|               | state.                            			      |	
 +---------------+-------------------------------------------------------------+
-| TAC           | Project is granted TAC representation.                      |
+| Incubation    | Project has matured beyond sandbox and may receive some     |
+|               | funding but does not yet have a representative on the TAC.  |
```

Comment:
```suggestion
|               | funding (while not impacting TAC projects).  The project also does not yet have a representative on the TAC.  |
```

Reply:
"some" funding is pretty vague...shall we have recommendations/rules to be more explicit here. 
Is there any board basic funding guidance/rules (even for TAC approved project, I assume these are Board decisions but I am not sure it is fully clear - conversation of budget allocated for project X on stat not really allocated)
Minimum or incubation funding would be maybe more realistic

Reply:
I might suggest removing "some" as neither that nor "minimum" is very specific.  The budget guidance below just says not to impact existing projects and I'm not sure we want to get more prescriptive, since budget is the board's responsibility.

Reply:
updated in next commit